test.cljs.edn

```diff
@@ -5,6 +5,13 @@
 
   ;; uncomment to launch tests in a headless environment
   ;; you will have to figure out the path to chrome on your system
-  :launch-js ["chromium" "--headless=new" "--disable-gpu" "--repl" :open-url]
+  :launch-js ["chromium"
+              "--headless=new"
+              "--disable-gpu"
+              "--no-sandbox"
```

Comment:
diables chromium sandboxing (most of it)

Reply:
So was Chromium sandboxing the root of all our issues?